blog/2024-07-10-uppy-4.0.md

```diff
@@ -318,11 +324,11 @@ new Uppy().use(XHR, {
 });
 ```
 
-Checkout the `@uppy/xhr-upload` [docs](/docs/xhr-upload/) for more info.
+Check out the `@uppy/xhr-upload` [docs](/docs/xhr-upload/) for more info.
 
 ## Simpler configuration for `@uppy/transloadit`
 
-To get started with `@uppy/transloadit` you would configure
+To get started with `@uppy/transloadit`, you would configure
 [`assemblyOptions`](/docs/transloadit/#assemblyoptions) with your auth key,
 template ID, and other optional values. `assemblyOptions` can be an object or a
 function, which is called per file, which returns an object:
```

Comment:
I'm a bit confused by the 2 x 'which' in this sentence. Which 'which' is referring to what?

Is this the best way to write this?